### PR TITLE
Removed unused stuff

### PR DESCRIPTION
**Pull request by Conni2461**
Not done. Need to go :) Finishing that later

### CODE REVIEW

File: lua/sql/defs.lua [hunk 1]
```diff
@@ -155,416 +146,28 @@ M.flags['open_privatecache']  = 0x00040000
...
-M.flags['dbconfig_enable_view']           = 1015
-M.flags['dbconfig_legacy_file_format']    = 1016
-M.flags['dbconfig_trusted_schema']        = 1017
-M.flags['dbconfig_max']                   = 1017
-
--- Authorizer Return Codes
-M.flags['deny']   = 1
-M.flags['ignore'] = 2
-
--- Authorizer Action Codes
```
Comment (tami5) [inline lua/sql/defs.lua:296]: Hmm just wondering what are these even for and when they do get return? is it important for our sql wrappers?

File: lua/sql/defs.lua [hunk 2]
```diff
@@ -155,416 +146,28 @@ M.flags['open_privatecache']  = 0x00040000
...
-M.flags['limit_variable_number']     = 9
-M.flags['limit_trigger_depth']       = 10
-M.flags['limit_worker_threads']      = 11
-
--- Prepare Flags
-M.flags['prepare_persistent'] = 0x01
-M.flags['prepare_normalize']  = 0x02
-M.flags['prepare_no_vtab']    = 0x04
-
--- Fundamental Datatypes
```
Comment (tami5) [inline lua/sql/defs.lua:357]: Hmm I remember coping those to sqlite_datatypes, I guess they will not be needed outside stmt.lua

File: lua/sql/defs.lua (hunk 1 above)
Comment (Conni2461) [reply lua/sql/defs.lua:296]: I think for `sqlite3_user_authenticate` or similar

Comment (Conni2461) [reply lua/sql/defs.lua:296]: No when doing int `sqlite3_set_authorizer`

Comment (Conni2461) [reply lua/sql/defs.lua:296]: https://sqlite.org/c3ref/c_alter_table.html

File: lua/sql/defs.lua (hunk 2 above)
Comment (Conni2461) [reply lua/sql/defs.lua:357]: Oh right. Do we need to keep those flags seem useful

File: lua/sql/defs.lua (hunk 1 above)
Comment (tami5) [reply lua/sql/defs.lua:296]: Oh, nice. I'm excited for a day we need suck a thing or a user do feature request :D

File: lua/sql/defs.lua (hunk 2 above)
Comment (tami5) [reply lua/sql/defs.lua:357]: I vote keep them, then later I'll remove sqlite_datatype

Comment (Conni2461) [reply lua/sql/defs.lua:357]: Added back

File: lua/sql/defs.lua (hunk 1 above)
Comment (Conni2461) [reply lua/sql/defs.lua:296]: We can always add stuff back 😆 But that are just the result codes so the function should still be callable.

**Pull request merged**